apis/cassandra/v1beta1/cdc_types.go

```diff
@@ -0,0 +1,50 @@
+package v1beta1
+
+// CDCConfiguration holds CDC config for the CassandraDatacenter. Note that it cannot contain arrays, channels, maps etc. because of the way the
+// reflection logic works which marshalls it into a string for the purposes of passing on the command line.
+type CDCConfiguration struct {
+	Enabled          bool    `json:"enabled"`
```

Comment:
That's a valid point. In k8ssandra-operator, I'd allow setting this `false` because a cluster level setting might need to be overridden for individual DCs. 

But in this case, cass-operator operates only at DC level, so I'm happy to make this change if you'd like?

Reply:
I have made this change as requested.